apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -65,6 +66,47 @@ const buildTokenExampleElement = (token, tokenType, tokenMetadata) => {
     return tokenExampleElementHandler[tokenType](token, tokenMetadata);
 };
 
+/**
+ * Builds the global token used element displayed primarily by alias tokens
+ * @param {string} globalToken the global token referenced by this alias token
+ * @returns a <span> HTML string containing the global token used
+ */
+const buildGlobalTokenUsedElement = globalToken => {
+    const globalTokenUsedElement = `
+    <span class="c-tokensTable-tokenDescription">
+      <span class="u-font-bold u-showAboveWide">Global token used:</span> 
+      <span class="c-tokensTable-token c-tokensTable-token--light">${globalToken}</span>
+    </span>`;
+
+    return deindentHTML(globalTokenUsedElement);
+};
+
+/**
+ * Builds the overall token description element for each type of token. The description content differs based on the type of token.
+ * @param {string} token the token value i.e. #000, #ffffff, #000|0.85 or #000000|0.85
+ * @param {*} tokenType the type of token i.e. color, spacing, radius
+ * @param {*} tokenMetadata the metadata for the token. data such as descriptions
+ * @returns {string} - the description HTML string
+ */
+const buildTokenDescriptionElement = (token, tokenType, tokenMetadata) => {
+    const tokenDescriptionElementHandler = {
```

Comment:
This variable name seems a little on the long side which makes it take a bit longer to grasp what's going on in the rest of the function. Would something like `handler` or `elementBuilder` suffice?

Reply:
I think `tokenTypeHandler` would be a mid way - I fear the other two examples are a little too vague personally! Will update